doc/guides/Dashboard.md

```diff
@@ -0,0 +1,216 @@
+# FOAM3 Dashboard Quick Start Guide
+
+This guide shows you how to create a simple dashboard in FOAM3 in just a few steps.
+
+## What You Need
+
+A FOAM3 dashboard requires these components:
+
+1. **Dashboard Class** - extends `foam.dashboard.view.DashboardView`
+2. **Widget Menus** - hidden menu entries that define your widgets
+3. **Citation Views** - for displaying individual records (optional)
+
+## Step 1: Create a Simple Dashboard
+
+Create `MyDashboard.js`:
+
+```javascript
+foam.CLASS({
+  package: 'my.dashboard',
+  name: 'MyDashboard',
+  extends: 'foam.dashboard.view.DashboardView',
+
+  properties: [
+    {
+      name: 'dashboardTitle',
+      value: 'My Dashboard'
+    },
+    {
+      name: 'main',
+      value: true
+    },
+    {
+      name: 'widgets',
+      factory: function() {
+        return {
+          'my.dashboard.users.table': { column: 6 },
```

Comment:
Size should be defined in menu.jrl as mentioned above. Columns also have sizes depending on screen width (mobile,desktop, etc) and can also use FR measurement. Can find a detailed explanation in foam3/src/foam/dashboard/view/DashboardView.js LN 86-111